<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="liberalsView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>561</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="facistView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>561</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="voteNo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Nein!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vote No</string>
    </property>
   </widget>
   <widget class="QPushButton" name="voteYes">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Ja!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vote Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vetoButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>VETO!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>450</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Veto</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="playersTurn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>REMOVE: ROLE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
